### PR TITLE
Bundle

### PR DESCRIPTION
**Pull request by vxsx**
Before CKEditor was asynchronously loading too many files to start
working. Some of those files could fail which would result in broken
editing area. If those files were cached that would result in broken text plugin

This PR bundles the files together and hashes the filename so new versions would always be cache-busted

Also it fixes a possible bug when user would override extraPlugins of ckeditor

### CODE REVIEW

File: djangocms_text_ckeditor/static/djangocms_text_ckeditor/js/cms.ckeditor.js [hunk 1]
```diff
@@ -55,17 +55,19 @@
                         settings: settings
                     }, this.options, options);
 
+                    // add extra plugins that we absolutely must have
+                    this.options.extraPlugins = this.options.extraPlugins += ',cmsplugins,cmswidget,cmsresize,' +
```
Comment (Brankub) [inline djangocms_text_ckeditor/static/djangocms_text_ckeditor/js/cms.ckeditor.js]: I'd prefer to make a line break after `=`

**Comment by Brankub** [comment]
overall looks good 👍

File: README.rst [hunk 2]
```diff
@@ -365,4 +365,20 @@ djangocms-text-ckeditor works well with `aldryn-search <https://github.com/aldry
 About CKEditor
 --------------
 
-The current integrated Version of CKeditor is **4.5.4**. For a full documentation visit: http://ckeditor.com/
+The current integrated Version of CKeditor is **4.5.9**. For a full documentation visit: http://ckeditor.com/
```
Comment (FinalAngel) [inline README.rst]: new one is 4.5.10

File: README.rst [hunk 3]
```diff
@@ -365,4 +365,20 @@ djangocms-text-ckeditor works well with `aldryn-search <https://github.com/aldry
...
+
+Building the js
```
Comment (FinalAngel) [inline README.rst]: write ``JavaScript``

File: README.rst [hunk 4]
```diff
@@ -365,4 +365,20 @@ djangocms-text-ckeditor works well with `aldryn-search <https://github.com/aldry
...
+---------------
+
+``djangocms-text-ckeditor`` distributes a javascript bundle required for the
+plugin to work, which contains CKEditor itself and all the necessary plugins for
+functioning within CMS. To build the bundle you need to have to install
+dependencies with ``npm install`` and then to run ``gulp bundle``.
+
+This command also updates the file name loaded based on the file contents.
+
```
Comment (FinalAngel) [inline README.rst]: add space

**Comment by FinalAngel** [comment]
👍

**Pull request merged**